test/aws/rails/sesv2_mailer_test.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'mail'
+
+class TestMailer < ActionMailer::Base
+  layout nil
+
+  def deliverable(options = {})
+    mail(
+      body: options[:body],
+      delivery_method: :sesv2,
+      from: options[:from],
+      subject: options[:subject],
+      to: options[:to],
+      cc: options[:cc],
+      bcc: options[:bcc]
+    )
+  end
+end
+
+module Aws
+  module Rails
+    describe Sesv2Mailer do
+      let(:client_options) do
+        {
+          stub_responses: {
+            send_email: {
+              message_id: ses_message_id
+            }
+          }
+        }
+      end
+
+      let(:mailer) { Sesv2Mailer.new(client_options) }
+
+      let(:sample_message) do
+        TestMailer.deliverable(
+          body: 'Hallo',
+          from: 'sender@example.com',
+          subject: 'This is a test',
+          to: 'recipient@example.com',
+          cc: 'recipient_cc@example.com',
+          bcc: 'recipient_bcc@example.com'
+        )
+      end
+
+      let(:ses_message_id) do
+        '0000000000000000-1111111-2222-3333-4444-555555555555-666666'
+      end
+
+      before do
+        ActionMailer::Base.add_delivery_method(:sesv2, Sesv2Mailer, client_options)
+      end
+
+      describe '#settings' do
+        it 'returns an empty hash' do
+          expect(mailer.settings).to eq({})
+        end
+      end
+
+      describe '#deliver!' do
```

Comment:
I think we'll want to test the other fields and transformations you are doing - specifically from_email_address (Rails 6 vs older), list_management_options, and configuration_set_name. Let's ensure good test coverage of this class.